spec/models/orcid_status_spec.rb

```diff
@@ -2,6 +2,19 @@
 require "rails_helper"
 
 RSpec.describe OrcidApiStatus, type: :model do
+  before do
+    stub_request(:get, "https://api.orcid.org/v3.0/apiStatus")
```

Comment:
Same here. This URL should be configurable too. It probably does not matter on the test since we are stubbing the request, but the `OrcidApiStatus` class itself should target the right API.